### DISCUSSION TITLE
关于图划分问题的请教

### DISCUSSION THREAD

**Discussion by DipsyZ** [alibaba/GraphScope#1875] [Q&A] [answered]
请问GraphScope中用户可以自定义图划分策略吗？
目前提供的划分策略有什么？如何配置

是否可以载入用户自定义的划分文件，不再划分直接进行图分析计算？
感谢🙏

**Comment by yecol** [comment, marked as answer]
Hi @DipsyZ ,

By default, GraphScope provides two partitioners, SegmentedPartitioner and HashPartitioner([code](https://github.com/alibaba/libgrape-lite/blob/master/grape/fragment/partitioner.h)).

GraphScope also supports customized partitioner, you may implement your own partitioner, 
and plug it in the right place ([ref](https://github.com/alibaba/GraphScope/search?q=SegmentedPartitioner)), it should work.
